<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePathsDialog</class>
 <widget class="QDialog" name="UpdatePathsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Paths</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="settingsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="algorithmLabel">
        <property name="text">
         <string>Matching Algorithm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="algorithmComboBox">
        <item>
         <property name="text">
          <string>Simple Ratio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Partial Ratio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Token Sort Ratio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Token Set Ratio</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="autoMatchButton">
        <property name="text">
         <string>Find Best Matches</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoMatchOnChangeCheckBox">
        <property name="text">
         <string>Auto-match on algorithm change</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sortByScoreCheckBox">
        <property name="text">
         <string>Sort by match score</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterVideoCheckBox">
        <property name="text">
         <string>Only show video files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
